src/components/program/SessionCard.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import type { Session } from "../../types/program.ts";
+import { dayAndTimeFormat } from "../../utils/dateformat.ts";
+import { useEffect, useState } from "react";
+
+interface Props {
+    session: Session;
+    clicked: boolean;
+    setClicked: (prev: (prev: boolean) => boolean) => void;
+}
+
+export const SessionCard = ({ session, clicked, setClicked }: Props) => {
+    const [isFavorite, setIsFavorite] = useState(false);
+
+    useEffect(() => {
+        const favoritesArray = JSON.parse(
+            localStorage.getItem("favorites") || "[]",
+        );
+        setIsFavorite(favoritesArray.includes(session.id));
+    }, [session.id, clicked]);
```

Comment:
Dette trenger du ikke en useEffect for, dette kan du heller legge inn i useState sin [initializer-funksjon](https://react.dev/reference/react/useState#avoiding-recreating-the-initial-state)

Reply:
Fant ikke en lur måte å løse dette på, skapte bugs 😅 ser mer på det i morgen

Reply:
Fikser i annen PR